docs/types/value-types.rst

```diff
@@ -221,7 +221,7 @@ Operators:
 
 .. warning::
     If you convert a type that uses a larger byte size to an ``address``, for example ``bytes32``, then the ``address`` is truncated.
-    To reduce conversion ambiguity version 0.4.24 and higher of the compiler force you make the truncation explicit in the conversion.
+    To reduce conversion ambiguity version 0.4.24 and higher versions of the Solidity compiler force you to make the truncation explicit in the conversion.
```

Comment:
```suggestion
    To reduce conversion ambiguity, starting with version 0.4.24 the compiler will force you to make the truncation explicit in the conversion.
```

I don't think we need to explicitly mention "solidity" in the solidity documentation :)